lib/mysqlDaoQueryHelper.js

```diff
@@ -261,73 +262,103 @@ class MysqlDaoQueryHelper {
 
         sqlObject = this._appendWhereClause(sqlObject, whereClause);
         return sqlObject;
     }
 
     /**
-     * @param {object} obj
-     * @param {object} [config]
-     * @param {boolean} [config.dontCleanMysqlFunctions] - defaults to true; includes CURRENT_TIMESTAMP and NOW();
-     *
-     * @returns {object} - cleans the values and returns an object with snake case props --> clean values
+     * @private
+     * @param {Object[]} objs
+     * @param {Boolean} [dontCleanMysqlFunctions] whether or not to allow MySQL functions to exist in values
+     * @returns {Object[]} - cleans the list of values and returns objects with snake_case props --> clean values
      */
-    _cleanAndMapValues(obj, config) {
-        const defaultConfig = {
-            dontCleanMysqlFunctions: true
-        };
+    _cleanAndMapValueList(objs , dontCleanMysqlFunctions = true) {
+        // Generate the snake-case property mappings for the first row only
+        const propertyMappings = MysqlDaoQueryHelper._getPropertyMappings(objs[0]);
+        return objs.map(obj => this.__cleanAndMapValues(obj, propertyMappings, dontCleanMysqlFunctions));
+    }
 
-        config = _.merge(defaultConfig, config);
+    /**
+     * @private
+     * @param {Object} obj
+     * @param {Boolean} [dontCleanMysqlFunctions] whether or not to allow MySQL functions to exist in values
+     * @returns {Object} - cleans the values and returns an object with snake_case props --> clean values
+     */
+    _cleanAndMapValues(obj, dontCleanMysqlFunctions = true) {
+        const propertyMappings = MysqlDaoQueryHelper._getPropertyMappings(obj);
+        return this.__cleanAndMapValues(obj, propertyMappings, dontCleanMysqlFunctions);
+    }
 
-        if (obj instanceof VO) {
+    /**
+     * @private
+     * @param {Object} obj
+     * @returns {Object} - a lookup object mapping property names to their snake_cased equivalents
+     */
+    static _getPropertyMappings(obj) {
+        if (typeof(obj.toJsObj) === 'function') {
             obj = obj.toJsObj();
-        } else if (typeof(obj.toJsObj) === 'function') {
+        } else if (typeof(obj.toJSON) === 'function') {
+            obj = obj.toJSON();
+        }
+
+        const mappings = {};
+        Object.keys(obj).forEach(key => {
+            mappings[key] = _.snakeCase(key);
+        });
+        return mappings;
+    }
+
+    /**
+     * Cleans values and maps them to snake_cased property names.
+     * This method should only be called by _cleanAndMapValues() or _cleanAndMapValueList()
+     * @private
+     * @param {object} obj
+     * @param {Object} propertyMappings maps property names to their snake_cased equivalends
+     * @param {Boolean} dontCleanMysqlFunctions whether or not to allow MySQL functions to exist in values
+     * @returns {object} - cleans the values and returns an object with snake_case props --> clean values
+     */
+    __cleanAndMapValues(obj, propertyMappings, dontCleanMysqlFunctions) {
```

Comment:
Minor: I think double-underscore functions usually imply a built-in variable/function (like `__dirname`). I'd keep private functions to one underscore.